docs/datastore/alloydb.md

```diff
@@ -47,9 +47,10 @@
 [install-alloydb-auth-proxy]: https://cloud.google.com/alloydb/docs/auth-proxy/connect#install
 
 
-## Enable private services access
+## Enable private services access (Non-Cloud Postgres)
```

Comment:
Why do we need to enable PSA for non-cloud postgres?

Reply:
Updated the commit 5f07874 with the changes suggested.